frontends/web/src/components/transactions/transaction.tsx

```diff
@@ -157,21 +157,34 @@ const Amounts = ({
   const conversion = amount?.conversions && amount?.conversions[defaultCurrency];
   const sign = getTxSign(type);
   const txTypeClass = `txAmount-${type}`;
+  const conversionPrefix = amount.estimated ? '\u2248' : false; // ≈
```

Comment:
a char or a bool is a bit weird for a type. null or undefined would be a better fit right?